app/Command/Compile.hs

```diff
@@ -55,7 +55,9 @@ compile :: PSCMakeOptions -> IO ()
 compile PSCMakeOptions{..} = do
   input <- globWarningOnMisses (unless pscmJSONErrors . warnFileTypeNotFound) pscmInput
   when (null input && not pscmJSONErrors) $ do
-    hPutStrLn stderr "psc: No input files."
+    hPutStrLn stderr $ unlines [ "purs compile: No input files."
```

Comment:
I think this will insert an extra line break; if we're using `unlines`, then I think the `hPutStrLn` should change to `hPutStr`